src/thor/dijkstras.cc

```diff
@@ -540,9 +462,10 @@ void Dijkstras::ExpandForwardMultiModal(GraphReader& graphreader,
     }
 
     // Add transfer time to the local time when entering a stop as a pedestrian. This
-    // is a small added cost on top of any costs along paths and roads
-    if (mode_ == TravelMode::kPedestrian) {
-      localtime += transfer_cost.secs;
+    // is a small added cost on top of any costs along paths and roads. We only do this
+    // once so if its from a transition we don't need to do it again
+    if (mode_ == TravelMode::kPedestrian && !from_transition) {
```

Comment:
`!from_transition` just curious, was this a bug you fixed?

Reply:
no its just a change. see previously localtime was a local variable which got recomputed every time. because we do the offset only once (ie before looking at transition nodes) and pass that into the transition expansions we need to protect against doing it again. previous code just always computed the offset twice to get the same answer. see above where `localtime` was declared before i removed it